cli/cephadm/cephadm.py

```diff
@@ -105,3 +105,17 @@ def logs(self, fsid, name):
         if isinstance(out, tuple):
             return out[0].strip()
         return out
+
+    def ceph_osd(self, id, conf, keyring):
+        """Run ceph-osd command
+
+        Args:
+            id (str): OSD id
+            conf (str): read configuration from the given confirguration file
+            keyring (str): keyring file name
+        """
+        cmd = f"{self.base_cmd} --id {id} --conf {conf} --keyring {keyring}"
```

Comment:
The command to be executed is ceph-osd right? But i dont see that in the cmd here
The self.base_cmd is cephadm. 
Please let me know if i missed anything

Reply:
@pranavprakash20 you are right, I miss that adding, thanks for catching it. 
I have corrected this now, and updated the full logs.